### PR TITLE
86 flyway implementation

### PR DESCRIPTION
**Pull request by ozdkaan**

### CODE REVIEW

File: src/main/resources/db/migration/V1__initial_db_creation.sql [hunk 1]
```diff
@@ -0,0 +1,1009 @@
...
+    script         varchar(1000)           not null,
+    checksum       integer,
+    installed_by   varchar(100)            not null,
+    installed_on   timestamp default now() not null,
+    execution_time integer                 not null,
+    success        boolean                 not null
+);
+
+create index if not exists flyway_schema_history_s_idx
+    on flyway_schema_history (success);
```
Comment (inciotiq) [inline src/main/resources/db/migration/V1__initial_db_creation.sql:1009]: Doesn't flyway create this table?

File: src/main/java/com/iotiq/application/config/DbMigrator.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    private DbMigrator() {
+    }
+
+    public static DbMigrator getInstance() {
+        if (instance == null) {
+            instance = new DbMigrator();
+        }
+        return instance;
+    }
```
Comment (inciotiq) [inline src/main/java/com/iotiq/application/config/DbMigrator.java:19]: Why are we implementing the singleton pattern here? Can you not use this as a component or service?

File: src/main/resources/db/migration/V1__initial_db_creation.sql (hunk 1 above)
Comment (ozdkaan) [reply src/main/resources/db/migration/V1__initial_db_creation.sql:1009]: Flayway creates this table, i removed

File: src/main/java/com/iotiq/application/config/DbMigrator.java (hunk 2 above)
Comment (ozdkaan) [reply src/main/java/com/iotiq/application/config/DbMigrator.java:19]: I refactored it as component

**Review by inciotiq — APPROVED** [review]
💯

**Pull request merged**